internal/component/prometheus/write/queue/types/network.go

```diff
@@ -2,6 +2,7 @@ package types
 
 import (
 	"context"
+	"github.com/grafana/alloy/syntax/alloytypes"
```

Comment:
This likely fails the linter, generally "core" golang imports are separated from other imports.

Reply:
It didnt..coolio :)